internal/database/db.go

```diff
@@ -194,41 +196,35 @@ func (dbs *SQLdb) GetHeaderForStableID(stableID string) (string, error) {
 }
 
 // MarkCompleted marks the file as "COMPLETED"
-func (dbs *SQLdb) MarkCompleted(file FileInfo, fileID int) error {
+func (dbs *SQLdb) MarkCompleted(file FileInfo, fileID, corrID string) error {
 	var (
 		err   error
 		count int
 	)
 
 	for count == 0 || (err != nil && count < dbRetryTimes) {
-		err = dbs.markCompleted(file, fileID)
+		err = dbs.markCompleted(file, fileID, corrID)
 		count++
 	}
 
 	return err
 }
 
 // markCompleted performs actual work for MarkCompleted
-func (dbs *SQLdb) markCompleted(file FileInfo, fileID int) error {
+func (dbs *SQLdb) markCompleted(file FileInfo, fileID, corrID string) error {
 	dbs.checkAndReconnectIfNeeded()
 
 	db := dbs.DB
-	const completed = "UPDATE local_ega.files SET status = 'COMPLETED', " +
```

Comment:
Maybe I should have thought of that in the DB rather PR, but is there any need for conversion here?

Reply:
(of the database)

Reply:
No, not since the query is switched from the legacy `local_ega` to the new `set_verified` function.